router/event.go

```diff
@@ -65,22 +65,25 @@ func (router *Router) eventFromRequest(r *http.Request) (*eventpkg.Event, string
 	}
 
 	event := eventpkg.New(eventType, mime, body)
+	if customevent, err := router.parseCustomEventAsCloudEvent(eventType, event.ContentType, body); err == nil {
```

Comment:
what's the point of having this error here if it's not used?

Reply:
It is used at the end of the if statement.